pandas/compat/__init__.py

```diff
@@ -451,3 +451,5 @@ def is_platform_mac():
 
 def is_platform_32bit():
     return struct.calcsize("P") * 8 < 64
+
+_default_fill_value = object()
```

Comment:
This probably isn't the right place for this, but it has to be importable from `pandas/arrays/base.py` and `core/algorithms.py`